CSD_PFC private count is not used

CSD_PFC.h defines PFCSuggestionIterator and PFCSuggestionIDIterator with private count members that are not used.

```
In file included from src/libdcs/CSD_PFC.cpp:33:
src/libdcs/CSD_PFC.h:191:9: warning: private field 'count' is not used [-Wunused-private-field]
        size_t count;
               ^
src/libdcs/CSD_PFC.h:328:9: warning: private field 'count' is not used [-Wunused-private-field]
        size_t count;
               ^
```

Compiler: Intel(R) oneAPI DPC++/C++ Compiler 2022.0.0 (2022.0.0.20211123)

--- a/libcds/include/libcdsTrees.h
+++ b/libcds/include/libcdsTrees.h
@@ -78,7 +78,7 @@ namespace cds_utils
 	}
 
 	/* writes e[p..p+len-1] = s, len <= W*/
-	inline void bitwrite (register uint *e, register uint p, register uint len, register uint s) {
+	inline void bitwrite (uint *e, uint p, uint len, uint s) {
 		e += p/W;
 		p %= W;
 		if (len == W) {

--- a/libcds/src/static/coders/huff.cpp
+++ b/libcds/src/static/coders/huff.cpp
@@ -139,8 +139,7 @@ namespace cds_static
 		return H;
 	}
 
-	void bitzero (register uint *e, register uint p,
-	register uint len) {
+	void bitzero (uint *e, uint p, uint len) {
 
 		e += p/W; p %= W;
 		if (p+len >= W) {

--- a/libcds/src/static/suffixtree/RMQ_succinct.cpp
+++ b/libcds/src/static/suffixtree/RMQ_succinct.cpp
@@ -104,7 +104,7 @@ namespace cds_static
 	unsigned int RMQ_succinct::log2fast(unsigned int v) {
 		unsigned int c = 0;		 // c will be lg(v)
 								 // temporaries
-		register unsigned int t, tt;
+		unsigned int t, tt;
 
 		if ((tt = v >> 16))
 			c = (t = v >> 24) ? 24 + LogTable256[t] : 16 + LogTable256[tt & 0xFF];

--- a/libcds/src/static/suffixtree/RMQ_succinct_lcp.cpp
+++ b/libcds/src/static/suffixtree/RMQ_succinct_lcp.cpp
@@ -103,7 +103,7 @@ namespace cds_static
 
 	uint RMQ_succinct_lcp::log2fast(uint v) {
 		uint c = 0;				 // c will be lg(v)
-		register uint t, tt;	 // temporaries
+	        uint t, tt;	 // temporaries
 
 		if ((tt = v >> 16))
 			c = (t = v >> 24) ? 24 + LogTable256[t] : 16 + LogTable256[tt & 0xFF];

--- a/libcds/src/static/suffixtree/factorization.cpp
+++ b/libcds/src/static/suffixtree/factorization.cpp
@@ -58,7 +58,7 @@ namespace cds_static
 		uint levelSizeAux[7]={0,0,0,0,0,0,0};
 		uint cont[7]={0,0,0,0,0,0,0};
 		listLength = l_Length;
-		register uint i;
+		uint i;
 		int j, k;
 		uint value, newvalue;
 		uint bits_BS_len = 0;
@@ -139,7 +139,7 @@ namespace cds_static
 
 	uint factorization::access(uint param) {
 		uint mult=0;
-		register uint j;
+		uint j;
 		uint partialSum=0;
 		uint ini=param-1;
 		uint * bsData = ((BitSequenceRG *)bS)->data;
@@ -173,7 +173,7 @@ namespace cds_static
 
 	uint factorization::access_seq(uint param, size_t *next_pos, bool dir) {
 		uint mult=0;
-		register uint j;
+	        uint j;
 		uint partialSum=0;
 		uint ini=param-1;
 		uint * bsData = ((BitSequenceRG *)bS)->data;

--- a/libcds/src/static/suffixtree/factorization_var.cpp
+++ b/libcds/src/static/suffixtree/factorization_var.cpp
@@ -41,7 +41,7 @@ namespace cds_static
 		uint *cont;
 		uint *contB;
 		listLength = l_Length;
-		register uint i;
+	        uint i;
 		int j, k;
 		uint value, newvalue;
 		uint bits_BS_len = 0;
@@ -171,7 +171,7 @@ namespace cds_static
 
 	uint factorization_var::access(uint param) {
 		uint mult=0;
-		register uint j;
+		uint j;
 		uint partialSum=0;
 		uint ini = param-1;
         unsigned char readByte;
@@ -200,7 +200,7 @@ namespace cds_static
 
 	uint factorization_var::access_seq(uint param, size_t *next_pos, bool dir) {
 		uint mult=0;
-		register uint j;
+		uint j;
 		uint partialSum=0;
 		uint ini = param-1;
         unsigned char readByte;

--- a/libhdt/src/huffman/huff.cpp
+++ b/libhdt/src/huffman/huff.cpp
@@ -162,8 +162,7 @@ namespace URICompressed{
         return H;
     }
 
-    void bitzero (register uint *e, register uint64_t p,
-    register uint len) {
+    void bitzero (uint *e, uint64_t p, uint len) {
 
         e += p/W; p %= W;
         if (p+len >= W) {

--- a/libhdt/src/libdcs/CSD_PFC.h
+++ b/libhdt/src/libdcs/CSD_PFC.h
@@ -188,7 +188,6 @@ class PFCSuggestionIterator: public hdt::IteratorUCharString {
 private:
 	CSD_PFC *pfc;
 	size_t max;
-	size_t count;
 	const char *prefix;
 	bool hasnext;
 	string tmpStr;
@@ -325,7 +324,6 @@ class PFCSuggestionIDIterator: public hdt::IteratorUInt {
 private:
 	CSD_PFC *pfc;
 	size_t max;
-	size_t count;
 	const char *prefix;
 	bool hasnext;
 	string tmpStr;

--- a/libhdt/src/libdcs/CSD_PFC.h
+++ b/libhdt/src/libdcs/CSD_PFC.h
@@ -201,7 +201,7 @@ private:
 public:
 
 	PFCSuggestionIterator(CSD_PFC *pfc, const char *prefix) :
-			pfc(pfc), count(1), prefix(prefix) {
+			pfc(pfc), prefix(prefix) {
 		max = pfc->getLength();
 		prefixlen = strlen(prefix);
 		hasnext = false;
@@ -337,7 +337,7 @@ private:
 public:
 
 	PFCSuggestionIDIterator(CSD_PFC *pfc, const char *prefix) :
-			pfc(pfc), count(1), prefix(prefix) {
+			pfc(pfc), prefix(prefix) {
 		max = pfc->getLength();
 		prefixlen = strlen(prefix);
 		hasnext = false;

